wgpu-hal/src/lib.rs

```diff
@@ -617,6 +617,10 @@ pub trait Queue: WasmNotSendSync {
 ///   live `CommandBuffers` built from it. All the `CommandBuffer`s
 ///   are destroyed, and their resources are freed.
 ///
+/// You may want to implement the [core::ops::Drop] trait to discard
+/// the current commands before the encoder is dropped (e.g. using
+/// [CommandEncoder::discard_encoding]).
```

Comment:
I'm not sure this note is correct - the user of the api shouldn't ever let a command encoder fall on the floor, they should all be calling https://wgpu.rs/doc/wgpu_hal/trait.Device.html#tymethod.destroy_command_encoder when it's time to delete the command encoder, correct?

Reply:
I also think this suggestion should be removed from the PR. From chat, my thinking was:

> The problem with this comment is that it neither tells the user of the trait that they can assume something is taken care of (say, "users can just drop `CommandEncoders`, implementations must handle that"), nor does it tell the implementor of the trait that something is taken care of ("Users must always pass this to `destroy_command_encoder`, implementors can just make sure that works and their job is done.").
>
> By just suggesting `Drop`, the comment muddies the waters about whose obligations are what. It's difficult to write code against this.